scripts/release/publishpackages.mjs

```diff
@@ -80,19 +80,6 @@ const tasks = new Listr( [
 					'ckeditor5'
 				]
 			} );
-		},
-		retry: 3
-	},
-	{
-		title: 'Checking if packages that returned E409 error code were uploaded correctly.',
-		task: async ( _, task ) => {
-			return releaseTools.verifyPackagesPublishedCorrectly( {
```

Comment:
What about this check? Are we not going to check for 409 errors any more?

Reply:
Review https://github.com/ckeditor/ckeditor5-dev/pull/1019, please.